server/routes/__tests__/homepage.spec.js

```diff
@@ -485,13 +507,53 @@ describe('GET /', () => {
         });
     });
 
+    it('renders the homepage with the correct number of update key info content tiles', () =>
+      request(app)
+        .get('/new-homepage')
+        .expect(200)
+        .then(response => {
+          const $ = cheerio.load(response.text);
+          expect($('#keyInfo a').length).toBe(4);
+        }));
+
+    it('Should render a html img tag in the homepage key info content tile', () =>
+      request(app)
+        .get('/new-homepage')
+        .expect(200)
+        .then(response => {
+          const $ = cheerio.load(response.text);
+          expect($('#keyInfo a div div').children()[0].type).toBe('tag');
+          expect($('#keyInfo a div div').children()[0].name).toBe('img');
+        }));
+
+    it('Should render a html h3 tag in the homepage key info content tile', () =>
+      request(app)
+        .get('/new-homepage')
+        .expect(200)
+        .then(response => {
+          const $ = cheerio.load(response.text);
+          expect($('#keyInfo a div div').children()[1].type).toBe('tag');
+          expect($('#keyInfo a div div').children()[1].name).toBe('h3');
+        }));
+
+    it('Should render a html h3 tag with the expected text in the homepage key info content tile', () =>
+      request(app)
+        .get('/new-homepage')
+        .expect(200)
+        .then(response => {
+          const $ = cheerio.load(response.text);
+          expect($('#keyInfo a div div h3:last').text()).toBe(
+            'A title that is long enough to be cropped with an ellipse added to the end',
```

Comment:
Personal preference, but this would be nicer as a const but meh:
```
const LAST_TILE_TITLE= 'A title that is long enough to be cropped with an ellipse added to the end'


// ...

expect($('#keyInfo a div div h3:last').text()).toBe(LAST_TILE_TITLE)
```

Reply:
I think in this case there is no real value from assigning the string to a variable. If this string was used in a number of tests then I would be setting it in a beforeEach and referencing the variable as you suggest. This string is used in a couple of the mock hubContent objects so there may be some value in doing this but prior to the hubContent being defined as apposed to within this test itself.